<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transacao</class>
 <widget class="QWidget" name="Transacao">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>78</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>78</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>401</width>
     <height>79</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="usuario1">
          <property name="text">
           <string>Usuario 1:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="usuario1_edit">
          <property name="text">
           <string>editU1</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="usuario2">
          <property name="text">
           <string>Usuario 2:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="usuario2_edit">
          <property name="text">
           <string>editU2</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="nome">
          <property name="text">
           <string>Nome:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nome_edit">
          <property name="text">
           <string>editNome</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="estado">
          <property name="text">
           <string>Estado</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="estado_edit">
          <property name="text">
           <string>editEstado</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>Deletar</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
